pkgs/servers/http/jboss/default.nix

```diff
@@ -1,7 +1,8 @@
 { lib, stdenv, fetchurl, jdk }:
 
 stdenv.mkDerivation {
-  name = "jboss-as-7.1.1.Final";
+  pname = "jboss-as";
+  version = "7.1.1.Final";
   src = fetchurl {
     url = "https://download.jboss.org/jbossas/7.1/jboss-as-7.1.1.Final/jboss-as-7.1.1.Final.tar.gz";
```

Comment:
Url could be templated here as well

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/123438/commits/a8f24f8ed468bf721e5a233401d9cf44c7da6412.